### PR TITLE
close tags by default

### PR DESCRIPTION
**Pull request by shrutimistry**
### Description

- added doc expansion to close tags
- added filter to search for tags
- sorted tags in alphabetical order

### How Has This Been Tested?
its not

### CODE REVIEW

File: src/core/index.js [hunk 1]
```diff
@@ -49,14 +49,14 @@ module.exports = function SwaggerUI(opts) {
...
     showExtensions: false,
+    tagsSorter: "alpha",
     supportedSubmitMethods: [
       "get",
       "put",
       "post",
       "delete",
       "options",
       "head",
-      "patch",
```
Comment (prees1) [inline src/core/index.js:59]: Was this intentional? I don't see it in the (amazing!) description in the pull request.

**Review by prees1 — CHANGES_REQUESTED** [review]
Looks like there are some changes to favicon files in this PR? `dist/tidal_icon.png`, `src/img/favicon-32x32.png`, src/plugins/topbar/favicon-32x32.png`. But don't see it mentioned in the description.

Also, great PR description! :)

File: src/core/index.js (hunk 1 above)
Comment (shrutimistry) [reply src/core/index.js:59]: yea since we don't use patch

**Review by prees1 — APPROVED** [review]
Great!

**Pull request merged**